int32 target_type  # 0: joint 1: pose
float64[] target_values
---
bool success
---
float64[] current_joint_values